discord/state.py

```diff
@@ -1637,13 +1637,20 @@ def parse_webhooks_update(self, data) -> None:
             )
             return
 
-        channel = guild.get_channel(int(data["channel_id"]))
-        if channel is not None:
-            self.dispatch("webhooks_update", channel)
+        channel_id = data["channel_id"]
```

Comment:
Should this not be `channel_id = data.get("channel_id")`?

Reply:
@Lulalaby

Reply:
it wasn't like that, i didn't want to change much

Reply:
data.get is not used in this file

Reply:
What was the original error? Was `channel_id` `null` (or 0) in the payload? Or was it completely missing from the payload?

Using `data.get` will handle both cases, but normal way will raise `KeyError` if `channel_id` was completely missing

Reply:
If the `channel_id` field is always present, `data["channel_id"]` should be used because it's faster. If the field is optional, only then it should be switched to `data.get("channel_id")`.

Reply:
That's the case. It exists but was null.